flags/conditions/strtobool.py

```diff
@@ -0,0 +1,14 @@
+def strtobool(val):
+    """Convert a string representation of truth to true (1) or false (0).
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+    """
+    val = val.lower()
+    if val in ("y", "yes", "t", "true", "on", "1"):
+        return 1
+    elif val in ("n", "no", "f", "false", "off", "0"):
+        return 0
+    else:
+        raise ValueError("invalid truth value %r" % (val,))
```

Comment:
One very minor thing: Could you move this file to `flags/utils.py`? I think I'd prefer it to live namespaced as a utility rather than within conditions.

Reply:
Done!